mem_mgmt/mem_mgmt.c

```diff
@@ -102,6 +106,10 @@ uint32_t mem_read(uint32_t modul, uint32_t *data)
 			length = CITIROC_LEN;
 			addr=(uint32_t *)(NVM_ADDR+CITIROC_OFFSET);
 			break;
+		case NVM_CITIROC_CONF_NO:
+            length=CITIROC_NO_LEN;
+            addr=(uint32_t *)(NVM_ADDR+CITIROC_CONF_OFFSET);
```

Comment:
This will probably not work, as `CITIROC_CONF_OFFSET` is not defined. (Check compiler warnings -- or errors -- much? :] ) Did you maybe mean `CITIROC_CONF_NO_OFFSET`?